pkg/tsdb/azuremonitor/azure-resource-graph-datasource.go

```diff
@@ -235,7 +235,7 @@ func (e *AzureResourceGraphDatasource) unmarshalResponse(res *http.Response) (Az
 
 	if res.StatusCode/100 != 2 {
 		azlog.Debug("Request failed", "status", res.Status, "body", string(body))
-		return AzureResourceGraphResponse{}, fmt.Errorf("request failed, status: %s, body: %s", res.Status, string(body))
+		return AzureResourceGraphResponse{}, fmt.Errorf("%s. Error returned from Azure: %s", res.Status, string(body))
```

Comment:
Slightly clearer the error comes from Azure - open to suggestions 🙂

Reply:
`Azure Resource Graph error: %s` ?